__tests__/endpoints.test.js

```diff
@@ -22,4 +23,23 @@ describe('GET/api/topics', ()=>{
             })
         })
     })
+})
+
+describe('GET/api', ()=>{
+    test('responds with 200 status code and an object describing all available endpoints', ()=>{
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body})=>{
+            const endpoints = body.endpoints;
+            const endpointsKeys = Object.keys(body.endpoints)
+            const testEndpointsKeys = Object.keys(testEndpoints)
+            expect(endpointsKeys).toEqual(testEndpointsKeys)
+            endpointsKeys.forEach((key)=>{
+                expect(typeof endpoints[key].description).toBe('string')
+                expect(Array.isArray(endpoints[key].queries)).toBe(true)
+                expect(typeof endpoints[key].exampleResponse).toBe('object')
+            })
```

Comment:
Another option for this would be to directly compare the two objects. One from the response and one required in here in this test file.